code/modules/reagents/holder.dm

```diff
@@ -811,3 +811,7 @@
 				random_reagents += R
 	var/picked_reagent = pick(random_reagents)
 	return picked_reagent
+
+/atom/movable/chem_holder
```

Comment:
why
why not just use a generic AM if obj doesn't suit for some reason?

Reply:
idk why tg did it like this lol